### PR TITLE
Extended QPT to athena-docdb

### PR DESCRIPTION
**Pull request by AbdulR3hman**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by aimethed — APPROVED** [review]

File: athena-docdb/src/main/java/com/amazonaws/athena/connectors/docdb/DocDBRecordHandler.java [hunk 1]
```diff
@@ -138,26 +141,38 @@ protected void readWithConstraint(BlockSpiller spiller, ReadRecordsRequest recor
             SOURCE_TABLE_PROPERTY, tableNameObj.getTableName());
 
         logger.info("Resolved tableName to: {}", tableName);
+        //Setup Project for Standard Query
+        String disableProjectionAndCasingEnvValue = configOptions.getOrDefault(DISABLE_PROJECTION_AND_CASING_ENV, "false").toLowerCase();
```
Comment (ejeffrli) [inline athena-docdb/src/main/java/com/amazonaws/athena/connectors/docdb/DocDBRecordHandler.java:145]: Why was this moved to here? Can we move it back to keep its declaration close to its usage?

File: athena-docdb/src/main/java/com/amazonaws/athena/connectors/docdb/qpt/DocDBQueryPassthrough.java
[the remark this answers is not in the document]
Comment (AbdulR3hman) [reply athena-docdb/src/main/java/com/amazonaws/athena/connectors/docdb/qpt/DocDBQueryPassthrough.java:1]: Yes; will do.

[the remark this answers is not in the document]
Comment (AbdulR3hman) [reply athena-docdb/src/main/java/com/amazonaws/athena/connectors/docdb/qpt/DocDBQueryPassthrough.java:1]: I've refactored the name and the schema parameters. the "arguments" is not needed like JDBC in this case; the interface method is enough.

File: athena-docdb/src/main/java/com/amazonaws/athena/connectors/docdb/DocDBRecordHandler.java (hunk 1 above)
Comment (AbdulR3hman) [reply athena-docdb/src/main/java/com/amazonaws/athena/connectors/docdb/DocDBRecordHandler.java]: done
